sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuReadCSVFileFormat.scala

```diff
@@ -41,7 +41,9 @@ class GpuReadCSVFileFormat extends CSVFileFormat with GpuReadFileFormatWithMetri
       filters: Seq[Filter],
       options: Map[String, String],
       hadoopConf: Configuration,
-      metrics: Map[String, GpuMetric]): PartitionedFile => Iterator[InternalRow] = {
+      metrics: Map[String, GpuMetric],
+      alluxionPathReplacementMap: Option[Map[String, String]] = None)
+  : PartitionedFile => Iterator[InternalRow] = {
```

Comment:
that is because the line length would be to long otherwise. we added the parameter alluxionPathReplacementMap. I think the style is supposed to be indented 2 from the function delcaraation though so I'll update